Examples/testCustomEntitlementsComputation/testCustomEntitlementsComputation/ContentView.swift

```diff
@@ -0,0 +1,278 @@
+                Task {
+                    await purchaseFirstOffering()
+                }
+            }
+            .font(.system(size: 20))
+            .foregroundColor(.white)
+            .frame(maxWidth: .infinity)
+            .padding()
+            .background(Color.blue)
+            .cornerRadius(20)
+            .padding()
+
+            .task {
+                Purchases.configureInCustomEntitlementsComputationMode(apiKey: Constants.apiKey,
+                                                                       appUserID: appUserID)
+                do {
+                    self.offerings = try await Purchases.shared.offerings()
+                    print("offerings: \(String(describing: offerings))")
+                }
+                catch {
+                    print("FAILED TO GET OFFERINGS: \(error.localizedDescription)")
+
+                }
+            }
+
+            NavigationView {
+                List(customerInfoStreamFires) { customerInfoStreamFire in
+                    NavigationLink(destination: CustomerInfoDetailsView(customerInfo: customerInfoStreamFire.customerInfo)) {
+
+                        VStack (alignment: .leading) {
+                            HStack {
+                                Text("Fired at:")
+                                    .bold()
+                                Text("\(customerInfoStreamFire.date, formatter: dateFormatter)")
+                            }
+                            HStack {
+                                Text("App User ID:")
+                                    .bold()
+                                Text(customerInfoStreamFire.customerInfo.originalAppUserId)
+                            }
+                        }
+                    }                }
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .principal) {
+                        VStack {
+                            Text("CustomerInfo stream values")
+                                .font(.headline)
+                                .foregroundColor(.primary)
+                                .padding()
+                            Text("List fills in below when new values arrive.")
+                                .foregroundColor(.primary)
+                        }
+                    }
+                }.task {
+                    subscribeToCustomerInfoStream()
+                }
+            }
+        }
+        .sheet(isPresented: $showingLogInAlert) {
+            CustomAlert(inputText: $appUserID, logIn: logIn)
+        }
+        .sheet(isPresented: $showingExplanation) {
+            ExplanationView()
+        }
+    }
+
+    func logIn(_ appUserID: String) {
+        print("Switching to user: \(appUserID)")
+        Task {
+            try await Purchases.shared.logIn(appUserID)
+        }
+    }
+
+    func subscribeToCustomerInfoStream() {
+        self.streamTask = Task {
```

Comment:
Maybe do this to ensure no errors are thrown:
```suggestion
        self.streamTask = Task<Void, Never> {
```